templates/base.html

```diff
@@ -56,22 +56,87 @@ <h1 class="brand m-0 h4"><a href="/">2021 Georgia Early Vote Ballot Tracker</a><
 
     <div id="content">
       {% block intro %}
-      <div class="container pt-5 mb-3">
-        <p class="m-0">
-          On January 5th there will be a statewide runoff election in Georgia. Let's get all of Georgia voting in this
-          important election! <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia" target="_blank" rel="noopener">Check which of your Facebook friends live in Georgia</a>, then use the search box below to look them up
-          and see if they have sent in their mail-in ballots.
-          <br><br>
-          All of this data comes from the <a href="https://elections.sos.ga.gov/Elections/voterabsenteefile.do">publicly
-          available files</a> published by the Georgia Secretary of State. If
-          you are wanting to track your own ballot status, consider utilizing
-          <a href="https://georgia.ballottrax.net/voter/">Georgia's official
-          site for this purpose</a>.
-        </p>
-      </div>
+        <!-- QUESTION: Why does my override for bg-danger not work but btn-danger does? -->
```

Comment:
i'm a little confused, where do you have bg-danager or btn-danger? i don't see either of them used outside the css file?

Reply:
we use `.btn-danger` in the footer and was wondering why i couldn't do the same with `.bg-danger`. but i just inline styled it...

![image](https://user-images.githubusercontent.com/1372946/101840043-4478cd00-3af8-11eb-8809-4d5f67684da9.png)